demosplan/DemosPlanCoreBundle/Command/Addon/AddonInstallFromZipCommand.php

```diff
@@ -503,4 +517,32 @@ function ($tag) {
 
         return $this->getHelper('question')->ask($input, $output, $question);
     }
+
+    private function getPathFromLocalDevelopment(InputInterface $input, SymfonyStyle $output, mixed $folder): string
+    {
+        $fs = new Filesystem();
+        $addonDevFolder = DemosPlanPath::getRootPath('addonDev');
+        if (!file_exists($addonDevFolder)) {
+            throw new RuntimeException("No folder {$folder} found. To develop addons locally, create a folder {$folder} and put your addons in there.");
```

Comment:
Maybe stranger things on my system happen, but I understand the code right that if I don't have a addonDev folder then this message will be printed to my console with addonDev instead of $folder right? During testing I had only the addonZips folder with four zips in it. But no addons and addonDev folder. So I expected this message: "No folder addonDev found. To develop addons locally, create a folder addonDev and put your addons in there.", Instead I had this message: "No folder addonZips found. To develop addons locally, create a folder addonZips and put your addons in there."

Reply:
did you provide `--develop` to the command @breierDemos? Looks as if you where in the "wrong" if branch

Reply:
@mussbach, yes I used bin/ewm dp:addon:install --develop. I think I know why this happen. With `$this->addOption('folder', '', InputOption::VALUE_REQUIRED, 'Folder to read addon zips from', 'addonZips');`  the default of `$folder` is "addonZips". So the default will be used.